src/csharp/Grpc.Tools/build/_grpc/_Grpc.Tools.targets

```diff
@@ -29,6 +31,101 @@
+            
+      <_ProtoNugetSearchPaths Include="@(_BothProtoNugetSearchPaths)" />
+      <_ProtoNugetSearchPaths Include="@(_ServerProtoNugetSearchPaths)" />
+      <_ProtoNugetSearchPaths Include="@(_ClientProtoNugetSearchPaths)" />
+    </ItemGroup>
+    <PropertyGroup>
+      <_FoundNugetProtoSearchPaths Condition="'@(_BothProtoNugetSearchPaths-&gt;Count())' &gt;= 1">True</_FoundNugetProtoSearchPaths>
+      <_FoundClientProtoFiles Condition="'@(_ClientProtoNugetFilesFound-&gt;Count())' &gt;= 1">True</_FoundClientProtoFiles>
+      <_FoundServerProtoFiles Condition="'@(_ServerProtoNugetFilesFound-&gt;Count())' &gt;= 1">True</_FoundServerProtoFiles>
+      <_FoundBothProtoFiles Condition="'@(_BothProtoNugetFilesFound-&gt;Count())' &gt;= 1">True</_FoundBothProtoFiles>
+      <_FoundAnyPackagedProtoFiles Condition="'$(_FoundClientProtoFiles)' == 'True' OR '$(_FoundServerProtoFiles)' == 'True' OR '$(_FoundBothProtoFiles)' == 'True'">True</_FoundAnyPackagedProtoFiles>
+    </PropertyGroup>
+    <Message Text="Searched for proto files in:" Importance="Normal" Condition="'$(_FoundNugetProtoSearchPaths)' == 'True'" />
+    <Message Text="	%(_ProtoNugetSearchPaths.FullPath)" Importance="Normal" Condition="'$(_FoundNugetProtoSearchPaths)' == 'True'" />
+    <Message Text="Including packaged protos with the following settings:" Importance="Low" Condition="'$(_FoundAnyPackagedProtoFiles)' == 'True'" />
+    <Message Text=" Protobuf_DefaultAccess: $(Proto_DefaultAccess)" Importance="Low" Condition="'$(_FoundAnyPackagedProtoFiles)' == 'True'" />
+    <Message Text=" Protobuf_DefaultClientBaseType: $(Proto_DefaultClientBaseType)" Importance="Low" Condition="'$(_FoundAnyPackagedProtoFiles)' == 'True'" />
+    <Message Text="Found @(_ClientProtoNugetFilesFound-&gt;Count()) packaged client proto(s):" Importance="Normal" Condition="'$(_FoundClientProtoFiles)' == 'True'" />
+    <Message Text="	%(_ClientProtoNugetFilesFound.Identity)" Importance="Normal" Condition="'$(_FoundClientProtoFiles)' == 'True'" />
+    <Message Text="Found @(_ServerProtoNugetFilesFound-&gt;Count()) packaged server proto(s):" Importance="Normal" Condition="'$(_FoundServerProtoFiles)' == 'True'" />
+    <Message Text="	%(_ServerProtoNugetFilesFound.Identity)" Importance="Normal" Condition="'$(_FoundServerProtoFiles)' == 'True'" />
+    <Message Text="Found @(_BothProtoNugetFilesFound-&gt;Count()) packaged client &amp; server proto(s):" Importance="Normal" Condition="'$(_FoundBothProtoFiles)' == 'True'" />
+    <Message Text="	%(_BothProtoNugetFilesFound.Identity)" Importance="Normal" Condition="'$(_FoundBothProtoFiles)' == 'True'" />
+  </Target>
+
+  <!-- 
+    This target will mark any Protobuf with GrpcPackAs set to Server, Client or Both as packable
+    and will set the path in the nuget package to one of the following:
+    protobuf/grpc
```

Comment:
using this convention seems very arbitrary. What makes you think that the uses will want to adopt this layout for their projects?

Reply:
The convention is for the layout of the nuget package, not the project itself (see screenshot in pr description)